Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -87,11 +92,27 @@ ErrorOr<void> JPEGLoadingContext::decode()
         cinfo.out_color_space = JCS_EXT_BGRX;
     }
 
+    auto marker = cinfo.marker_list;
+    while (marker) {
+        if (marker->marker == jpeg_app1 && marker->data_length >= 6) {
+            JOCTET expected[6] = { 'E', 'x', 'i', 'f', 0, 0 };
+
+            if (memcmp(marker->data, expected, 6) != 0) {
```

Comment:
Generally we try and avoid things like `memcmp()` and use something more type-safe instead. However, this looks correct to me, and image decoding is not at all my area so maybe there isn't a nicer way to do it at the moment.

Reply:
```c++
static constexpr auto expected = "Exif\0\0"sv;
auto signature = StringView { marker->data, 6 };
if (signature != expected) {
	[...]
}
```
That should work!